### PR TITLE
db/climate-marker

### PR DESCRIPTION
**Pull request by bre1470**
If we go with this design, we should reduce the number of cities.

### CODE REVIEW

File: samples/dashboard/demos/climate/demo.js [hunk 1]
```diff
@@ -120,10 +120,27 @@ async function setupDashboard() {
...
+                        verticalAlign: 'top',
+                        x: 5,
+                        y: -10
+                    }, {
+                        crop: false,
+                        enabled: true,
+                        formatter: function () {
+                            const point = this;
+
+                            return labelFormatter(point.y, true);
```
Comment (pawelfus) [inline samples/dashboard/demos/climate/demo.js:138]: Nitpicking: unnecessary extra argument

**Review by pawelfus — APPROVED** [review]

**Comment by TorsteinHonsi** [comment]
> If we go with this design, we should reduce the number of cities.

I don't think we need to do that. The data labels handle overlapping gracefully. Instead, I think we should
* Use true data label mapmarkers as seen in the `maps/demo/mappoint-datalabels-mapmarker` demo. The difference is that the marker _is_ the label itself, instead of the label being superimposed on the point marker. The advantage being handling of label overlapping and, optionally, nice stacking of labels if `allowOverlap` is true.
* Show Celcius instead of Kelvin. That way we have mostly two digits and can...
* Make the labels smaller.

See the [Netatmo Weathermap](https://weathermap.netatmo.com/) for inpiration. Overlapping of labels isn't a problem.

File: samples/dashboard/demos/climate/demo.js (hunk 1 above)
Comment (bre1470) [reply samples/dashboard/demos/climate/demo.js:138]: Good catch.

**Review by TorsteinHonsi — CHANGES_REQUESTED** [review]
Remaining change requests:
* Show Celcius instead of Kelvin. That way we have mostly two digits and can...
* Make the labels smaller.

**Comment by TorsteinHonsi** [comment]
This can be avoided by setting ordering the points by descending latitude.
Skjermbilde 2023-01-04 kl  13 38 09

**Review by karolkolodziej — CHANGES_REQUESTED** [review]
After changing the navigator, the text inside the markers is not centered.
image

**Comment by bre1470** [comment]
Yes, I noticed a lot is going on there.
- The styling fix unfortunately does not work for label shapes.
- The animation of label shapes has some position flaw, which makes them move with every redraw.
- In addition the border of the selected city is gone.
- Also the styling fix does not cover selected cities.

**Comment by bre1470** [comment]
> * Also the styling fix does not cover selected cities.

Pending PR: https://github.com/highcharts/highcharts/pull/18247

File: samples/dashboard/demos/climate/demo.css [hunk 2]
```diff
@@ -64,12 +67,19 @@
     stroke: #986;
 }
 
-#world-map .highcharts-series-1 .highcharts-data-label text {
+#world-map .highcharts-series-1 .highcharts-data-label:nth-child(odd) text {
```
Comment (sebastianbochan) [inline samples/dashboard/demos/climate/demo.css:70]: We can use default style for text and use only `even` as extension. We do not need to declare odd and even.

**Review by sebastianbochan — APPROVED** [review]

**Pull request merged**